pose_commander:
  ros__parameters:
    linear_velocity: 0.2 # METERS PER SECOND
    angular_velocity: 0.5 # RADIANS PER SECOND
